src/test/java/pyke/PykeTest.java

```diff
@@ -0,0 +1,12 @@
+package pyke;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
```

Comment:
If I'm not wrong, static imports should be separated from normal ones (with static arranged at the top). The checkstyle rules given to us should accept it in this format

```java
import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.fail;

import org.junit.jupiter.api.Test;
```